docs/plugins/testing_workshop_docs/index.md

```diff
@@ -0,0 +1,8 @@
+# Testing workshop articles 
+
+This is the frontpage for the articles taken from the [January 2022 workshop on testing](https://youtu.be/IsHYnI8Tbfw?list=PLilvrWT8aLuYID3YZ7KddS5ky2SaH4DKK). These articles will be more meaningful to you if you are familiar with the Python programming language and napari software. These articles summarize the information in the video and should stand on their own. The articles are listed here with the timings on the video next to the article:  
+* Article 1: [Python’s assert keyword](./article-1-pythons-assert-keyword.md) - starts at minute [5:23](https://youtu.be/IsHYnI8Tbfw?list=PLilvrWT8aLuYID3YZ7KddS5ky2SaH4DKK&t=333).  
+* Article 2: [Pytest testing framework](./article-2-pytest-testing-frameworks.md) - starts at minute [9:21](https://youtu.be/IsHYnI8Tbfw?list=PLilvrWT8aLuYID3YZ7KddS5ky2SaH4DKK&t=561).  
+* Article 3: [Readers and fixtures](./article-3-readers-and-fixtures.md) - starts at minute [15:42](https://youtu.be/IsHYnI8Tbfw?list=PLilvrWT8aLuYID3YZ7KddS5ky2SaH4DKK&t=942).  
+* Article 4: [Test coverage](./article-4-test-coverage.md) - starts at minute [28:26](https://youtu.be/IsHYnI8Tbfw?list=PLilvrWT8aLuYID3YZ7KddS5ky2SaH4DKK&t=1706).  
```

Comment:
I was not a part of the initial proposal but do we want to keep this wording and call these articles? I'm not against it, but they do stand out as very different from the rest of the documentation. @DragaDoncila maybe you have context on this?

Reply:
@melissawm no I think it's a good point that we don't use the `Article` phrasing anywhere else. Since this is kind of a progressive tutorial I wonder if we could just remove the word `Article` so it's number + article name?

Reply:
Just keep the number and the title.